tests/DFE.FindInformationAcademiesTrusts.CypressTests/cypress/e2e/regression/trusts/contacts-page.cy.ts

```diff
@@ -64,6 +64,26 @@ describe("Testing the components of the Trust contacts page", () => {
                     .checkSuccessPopup('Changes made to the SFSO (Schools financial support and oversight) lead name and email were updated')
                     .checkErrorPopupNotPresent();
             });
+
+            it(`Checks that when cancelling the edit of a TRM contact that I am taken back to the previous page and that entered data is not saved`, () => {
+                contactsPage
+                    .editTrustRelationshipManager("Should Notbe Seen", "exittest@education.co.uk")
```

Comment:
This page object model method presses the save button so this test should fail. We should make a new method which stops before pressing the save button